### PR TITLE
Update FW 4.85 Latest Version

### PR DESCRIPTION
**Pull request by Joaozin-tech**
Updates the latest FW version according to: https://www.playstation.com/en-us/support/system-updates/ps3/
* What's New in Version 4.85

This system software update improves the quality of system performance.

### CODE REVIEW

**Comment by AniLeo** [comment]
Still shows 4.84 for most regions, should wait until the update is actually out for most regions

**Comment by MSuih** [comment]
We can keep the pull request open for now and merge it when enough time has passed.

File: rpcs3/rpcs3qt/main_window.cpp [hunk 1]
```diff
@@ -578,7 +578,7 @@ void main_window::InstallPup(const QString& dropPath)
 		version_string.erase(version_pos);
 	}
 
-	const std::string cur_version = "4.84";
+	const std::string cur_version = "4.85.1";
```
Comment (MsDarkLow) [inline rpcs3/rpcs3qt/main_window.cpp]:
```suggestion
	const std::string cur_version = "4.85";
```
Shouldn't this be 4.85? I don't seem to find this .1 anywhere. And the update page specifically says 4.85

**Comment by aakh1361** [comment]
for me show 4.85
"An update to the PS3™ system software was released on August 28, 2019. You can use this update to upgrade your system software to version 4.85."

**Comment by Parovozik** [comment]
Another useless ps3 firmware update from caring Sony, which doesn’t come out so that something works better for you. Real technical support has long been discontinued XD

**Comment by anseesaw** [comment]
TIM截图20190902184807

**Pull request merged**